### PR TITLE
update for julia v0.6

### PR DESCRIPTION
**Pull request by mcreel**
Updated to work with julia 0.6.x. The plot is commented out for now.

### CODE REVIEW

File: examples/regression-example.jl [hunk 1]
```diff
@@ -6,11 +6,11 @@ the predictions from the trained net.
 =#
 using MXNet
 using Distributions
-using Plots
+#using Plots
 
 # data generating process
 generate_inputs(mean, var, size) = rand(MvNormal(mean, var), size)
-output(data) = sin(data[1,:]).*sin(data[2,:])./(data[1,:].*data[2,:])
+output(data) = sin.(data[[1],:]).*sin.(data[[2],:])./(data[[1],:].*data[[2],:])
```
Comment (vchuravy) [inline examples/regression-example.jl]: Why is the additional `[1]` in `data[[1],:]` necessary?

**Comment by mcreel** [comment]
data is a two dimensional array. data[1,:] results in a one dimensional
array. This difference leads to an error at some point. data[[i],:] is
still two dimensional, and this avoids the problem. Perhaps there's  a
better way to do this, though.

<http://www.uab.cat/>

Michael Creel
*UAB, BGSE, MOVE*
Director, Dept. d'Economia I d'Història Econòmica


T +34 93 581 1696
web page <http://pareto.uab.es/mcreel>

**Comment by vchuravy** [comment]
I think `data[1:1, :]` is the canonical way (and doesn't require the allocation of a new array)

**Comment by mcreel** [comment]
This changes the use of indices following the discussion

**Comment by mcreel** [comment]
Thanks, I still haven't adjusted fully to the new scheme. I have modified
the PR.

<http://www.uab.cat/>

Michael Creel
*UAB, BGSE, MOVE*
Director, Dept. d'Economia I d'Història Econòmica


T +34 93 581 1696
web page <http://pareto.uab.es/mcreel>

**Pull request merged**